### DISCUSSION TITLE
Custom Tokens Instead of Overriding

### DISCUSSION THREAD

**Discussion by funisfun8** [gmsgowtham/react-native-marked#552] [Q&A]
Hello there,
I am working on a React Native project that would benefit greatly from comprehensive markdown parsing. However, the application may require custom markdown syntax for non-standard elements. I see that this library allows for overriding built-in elements, such as the LATEX example in the readme, but does it allow for additional custom elements such as are described [here](https://marked.js.org/using_pro#custom-extensions-example)?
Thank you for your work on the library. I hope to use it soon.

**Comment by gmsgowtham** [comment]
@funisfun8 Tokenizer can be used to generate custom token which can be rendered via Custom Renderer. ref: https://github.com/gmsgowtham/react-native-marked?tab=readme-ov-file#using-tokenizer-with-custom-components
